modules/platform/nuxeo-apidoc-server/nuxeo-apidoc-repo/src/test/java/org/nuxeo/apidoc/test/TestSnapshotPersist.java

```diff
@@ -125,61 +145,86 @@ protected String represent(NuxeoArtifact artifact) {
         return res;
     }
 
-    protected void checkBundles(DistributionSnapshot snapshot) throws IOException {
+    protected void checkBundles(DistributionSnapshot snapshot, boolean partial) throws IOException {
         List<String> bids = snapshot.getBundleIds();
 
         StringBuilder sb = new StringBuilder();
         bids.forEach(bid -> sb.append(represent(snapshot.getBundle(bid))));
```

Comment:
If you want you could build the whole string directly as:

    String s = bids.stream().map(snapshot::getBundle).map(this::represent).collect(Collectors.joining());